riscv/src/compiler.rs

```diff
@@ -1128,10 +1173,17 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             vec![format!("jump_and_link_dyn {rs};")]
         }
         "call" => {
-            if let [label] = args {
-                vec![format!("call {};", argument_to_escaped_symbol(label))]
-            } else {
-                panic!()
+            // Depending on what symbol is called, the call is replaced by a
+            // powdr asm call, or a call to a coprocessor if a special function
+            // has been recognized.
+            match args {
+                [Argument::Expression(Expression::Symbol(label))]
+                    if let Some(replacement) = has_coprocessor_substitution(label) =>
+                {
+                    vec![replacement.to_string()]
+                }
+                [label] => vec![format!("call {};", argument_to_escaped_symbol(label))],
+                _ => panic!(),
```

Comment:
the `if let` guard is why I have to add a nightly feature and so why CI breaks. I can replace it with an uglier sub-`match`, of course. Still, I am checking if adding this nightly feature instead would be ok?

Reply:
That's unfortunate but I think we should not add a nightly feature.

Reply:
you break my heart